frontend/README.md

```diff
@@ -10,6 +10,10 @@ The UI part of the system that intends to provide user-friendly wrapper over the
 
 Please refer to the root readme to learn more about general development setup.
 
+### Environment variables
+
+- `API_ORIGIN` (optional, default `/api`): relative path or the url under which `api` service is running. For example, if you run api on port 4000 via `PORT=4000 npm run start`, the value should be `http://localhost:4000`
```

Comment:
This is a bit odd.

If the path is relative, why is the example providing the full url?

Reply:
It says "relative path or the url". Or what would be your suggestion?

Reply:
- providing two examples,
- removing PORT env variable out of the equation here (since it's still documented in different place, we don't need to mention it here in the example)